python/az/aro/azext_aro/custom.py

```diff
@@ -138,43 +150,47 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
 
 def aro_delete(cmd, client, resource_group_name, resource_name, no_wait=False):
     # TODO: clean up rbac
-
     try:
         oc = client.get(resource_group_name, resource_name)
-
-        master_subnet = oc.master_profile.subnet_id
-        worker_subnets = {w.subnet_id for w in oc.worker_profiles}
-
-        master_parts = parse_resource_id(master_subnet)
-        vnet = resource_id(
-            subscription=master_parts['subscription'],
-            resource_group=master_parts['resource_group'],
-            namespace='Microsoft.Network',
-            type='virtualNetworks',
-            name=master_parts['name'],
-        )
+        resources = get_ocp_cluster_network_resources(cmd.cli_ctx, oc)
```

Comment:
nit: `get_cluster_network_resource`.  I think there is a thin line when naming is over-expressive :)

Reply:
Add comment to the reader what we are doing this block